config/routes.rb

```diff
@@ -63,7 +63,7 @@
     patch "/", to: "dashboard#update"
     get "/items", to: "items#index"
     get "/orders/:id", to: "orders#show"
-    patch "/items", to: "items#deactivate"
+    patch "/items", to: "items#update"
```

Comment:
This is much more RESTful. Great call!

Reply:
AGREED! I may snag this refactor to change the Admin::MerchantController methods (from disable/enable to update)